dashboard/app/views/census_reviewers/review_reported_inaccuracies.html.haml

```diff
@@ -0,0 +1,8 @@
+- script_data = {}
+- script_data[:authenticityToken] = form_authenticity_token
+- inaccuracy_reports = Census::CensusSubmission.left_joins(:census_inaccuracy_investigations).where(inaccuracy_reported: true).where('census_inaccuracy_investigations.id is null')
+- script_data[:reportsToReview] = inaccuracy_reports.map(&:inaccuracy_review_data).flatten.compact.map(&:to_json)
+%script{src: minifiable_asset_path('js/census_reviewers/review_reported_inaccuracies.js'), data: {props: script_data.to_json}}
+
+#application-container
+  -# populated by React
```

Comment:
nit: it's not required, but we prefer to have a method for each action in the controller, even if it's empty just to it's more clear & discoverable. 

Beyond that, the view should not be performing logic like Sql queries. This should go in the controller, and populate a `@member` variable which will be consumed in the view.

Reply:
Changed this to put the logic in the controller and thus also have a method for the action (which is not empty.)